frontend/public/components/storage-class-form.tsx

```diff
@@ -825,6 +777,10 @@ export class StorageClassForm_ extends React.Component<
     return isRequired;
   };
 
+  handleComponentParams = (value: string, key: string) => {
+    this.setParameterHandler(key, value, false);
+  };
```

Comment:
```suggestion
  handleComponentParams = (value: string, key: string) => this.setParameterHandler(key, value, false);
```

Reply:
since this is only used in one place. We could also use it directly. but it's up to you.